cluster/components/kuberay/README.md

```
@@ -0,0 +1,94 @@
+    # rayservice-sample-head-svc                    ClusterIP   10.96.34.90     <none>        10001/TCP,8265/TCP,52365/TCP,6379/TCP,8080/TCP,8000/TCP   4m58s
+    # rayservice-sample-raycluster-6mj28-head-svc   ClusterIP   10.96.171.184   <none>        10001/TCP,8265/TCP,52365/TCP,6379/TCP,8080/TCP,8000/TCP   6m21s
+    # rayservice-sample-serve-svc                   ClusterIP   10.96.161.84    <none>        8000/TCP                                                  4m58s
+    ```
+
+**Setting up Dashboard Ingress:**
+
+1. Apply the ingress to expose dashboard to public users `kubectl apply -f ingress.yaml` (please update the host as needed in ingress.yaml, it is assumed that Kong has been setup)
+
+**Send requests to the Serve applications by the Kubernetes serve service**
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)